src/stylesheets/blog.scss

```
@@ -181,19 +379,24 @@ $tablet: "screen and (max-width : 900px)";
   }
 
   .blog-title-link {
-    transition: color cubic-bezier(0.4, 0, 0.2, 1) 0.2s;
-    color: #1a1a1a;
     text-decoration: none;
-    &:hover {
-      color: #ffe100;
-    }
   }
 
-  .blog-title {
+  .blog-title, .featured-blog-title {
     font-size: 28px;
     line-height: 1.5;
     font-weight: 700;
     margin: 0;
+    background: #1a1a1a;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    transition: color cubic-bezier(0.4, 0, 0.2, 1) 0.2s;
+
+    &:hover {
+      background: linear-gradient(87.89deg, #e69d00 0%, #e6b000 100%);
+      -webkit-background-clip: text;
+      -webkit-text-fill-color: transparent;
+    }
```

```suggestion
.blog-title, .featured-blog-title {
    font-size: 28px;
    line-height: 1.5;
    font-weight: 700;
    margin: 0;
    background: linear-gradient(87.89deg, #e69d00 0%, #e6b000 100%);
    -webkit-background-clip: text;
    background-clip: text;
    color:rgba(0,0,0,1);
    transition: color cubic-bezier(0.4, 0, 0.2, 1) 0.2s;

    &:hover {
      color:rgba(0,0,0,0);
    }
```
Currently, the transition doesn't work, removing `-webkit-text-fill-color: transparent` could fix it.